cterasdk/edge/volumes.py

```diff
@@ -66,6 +66,31 @@ def add(self, name, size=None, filesystem='xfs', device=None, passphrase=None):
 
         return response
 
+    def modify(self, name, size=None):
```

Comment:
This is a bit more tricky. We can't update a volume's size by changing its config. The size is tied to the underlying storage presented to our virtual appliance. All we can really modify is the volume name.
We can't change the file-system type or apply encryption after the fact.

Reply:
First, I tried it and you can modify the size as long as it is larger than the current one and not larger than the drive size. I thought if I should add any protections, but decided to let the backend fail if the user tries to do something that does not work.
As for the name, currently the name if the identifier of the volume, so I didn't implement the ability to change it (mainly because I didn't implement it in the Ansible part).